examples/csharp/AsyncSuffix/Greet/generate_proto.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Generate without async suffix
+protoc --csharp_out=./Server/ --grpc_out=./Server/ --plugin=protoc-gen-grpc=grpc_csharp_plugin greet.proto
+
+# Generate with async suffix
+protoc --csharp_out=./Server/ --grpc_out=./Server/ --plugin=protoc-gen-grpc=grpc_csharp_plugin --grpc_opt=append_async_suffix=true greet.proto
```

Comment:
since this directory is focused on the async suffix option, let's just generate the protos with the async suffix option